src/EPR.Calculator.API/Enums/RunClassification.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace EPR.Calculator.API.Enums
```

Comment:
Please replace this 
`    public enum RunClassiciations
    {
        [Description("IN THE QUEUE")]
        INTHEQUEUE = 1,
        [Description("RUNNING")]
        RUNNING,
        [Description("UNCLASSIFIED")]
        UNCLASSIFIED,
        [Description("PLAY")]
        PLAY,
        [Description("ERROR")]
        ERROR
    }`

Reply:
resolved the comment.